doc/2/controllers/bucket/create/index.md

```diff
@@ -0,0 +1,91 @@
+---
+code: true
+type: page
+title: create
+---
+
+# create
+
+Creates a S3 bucket
+
+## Query Syntax
+
+### HTTP
+
+```http
+URL: http://kuzzle:7512/_plugin/s3/bucket/create
```

Comment:
bucketName and bucketRegion are args, they should appear in url but I'm not sure how ?

Reply:
I got it now